src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -219,6 +219,9 @@
     <value>The supplied value for "{0}" paramter is invalid.</value>
     <comment>{0} is parameter name</comment>
   </data>
+  <data name="UnsupportedPrettyParameter" xml:space="preserve">
+    <value>The requested "_pretty" parameter is not supported.</value>
+  </data>
```

Comment:
Probably need to add a comment of `{NumberedPlaceHolder="_pretty"}` so that it wouldn't be translated.

Reply:
Thanks! Added in fccf656.